bash_loading_animations.sh

```diff
@@ -1,145 +1,86 @@
+declare BLA_breathe=( 0.9 '  ()  ' ' (  ) ' '(    )' ' (  ) ' )
+declare BLA_growing_dots=( 0.5 '.  ' '.. ' '...' '.. ' '.  ' '   ' )
+declare BLA_passing_dots=( 0.25 '.  ' '.. ' '...' ' ..' '  .' '   ' )
+declare BLA_metro=( 0.2 '[    ]' '[=   ]' '[==  ]' '[=== ]' '[ ===]' '[  ==]' '[   =]' )
 
 ## UTF-8 animations ##
 # Require Unicode support (will work in most modern terminals, but not in TTY).
 # Some animations may not render properly with certain fonts.
-classic_utf8=( 0.25 '—' '\' '|' '/' )
-bounce=( 0.3 . · ˙ · )
-vertical_block=( 0.25 ▁ ▂ ▃ ▄ ▅ ▆ ▇ █ █ ▇ ▆ ▅ ▄ ▃ ▁ )
-horizontal_block=( 0.25 ▏ ▎ ▍ ▌ ▋ ▊ ▉ ▉ ▊ ▋ ▌ ▍ ▎ ▏ )
-quarter=( 0.25 ▖ ▘ ▝ ▗ )
-triangle=( 0.45 ◢ ◣ ◤ ◥)
-semi_circle=( 0.1 ◐ ◓ ◑ ◒ )
-rotating_eyes=( 0.4 ◡◡ ⊙⊙ ⊙⊙ ◠◠ )
-firework=( 0.4 '⢀' '⠠' '⠐' '⠈' '*' '*' ' ' )
-braille=( 0.2 ⠁ ⠂ ⠄ ⡀ ⢀ ⠠ ⠐ ⠈ )
-braille_whitespace=( 0.2 ⣾ ⣽ ⣻ ⢿ ⡿ ⣟ ⣯ ⣷ )
-trigram=( 0.25 ☰ ☱ ☳ ☶ ☴ )
-arrow=( 0.15 ▹▹▹▹▹ ▸▹▹▹▹ ▹▸▹▹▹ ▹▹▸▹▹ ▹▹▹▸▹ ▹▹▹▹▸ ▹▹▹▹▹ ▹▹▹▹▹ ▹▹▹▹▹ ▹▹▹▹▹ ▹▹▹▹▹ ▹▹▹▹▹ ▹▹▹▹▹ )
-bouncing_ball=( 0.4 '(●     )' '( ●    )' '(  ●   )' '(   ●  )' '(    ● )' '(     ●)' '(    ● )' '(   ●  )' '(  ●   )' '( ●    )' )
-big_dot=( 0.7 ∙∙∙ ●∙∙ ∙●∙ ∙∙● )
-modern_metro=( 0.15 ▰▱▱▱▱▱▱ ▰▰▱▱▱▱▱ ▰▰▰▱▱▱▱ ▱▰▰▰▱▱▱ ▱▱▰▰▰▱▱ ▱▱▱▰▰▰▱ ▱▱▱▱▰▰▰ ▱▱▱▱▱▰▰ ▱▱▱▱▱▱▰ ▱▱▱▱▱▱▱ ▱▱▱▱▱▱▱ ▱▱▱▱▱▱▱ ▱▱▱▱▱▱▱ )
-pong=( 0.35 '▐⠂       ▌' '▐⠈       ▌' '▐ ⠂      ▌' '▐ ⠠      ▌' '▐  ⡀     ▌' '▐  ⠠     ▌' '▐   ⠂    ▌' '▐   ⠈    ▌' '▐    ⠂   ▌' '▐    ⠠   ▌' '▐     ⡀  ▌' '▐     ⠠  ▌' '▐      ⠂ ▌' '▐      ⠈ ▌' '▐       ⠂▌' '▐       ⠠▌' '▐       ⡀▌' '▐      ⠠ ▌' '▐      ⠂ ▌' '▐     ⠈  ▌' '▐     ⠂  ▌' '▐    ⠠   ▌' '▐    ⡀   ▌' '▐   ⠠    ▌' '▐   ⠂    ▌' '▐  ⠈     ▌' '▐  ⠂     ▌' '▐ ⠠      ▌' '▐ ⡀      ▌' '▐⠠       ▌' )
-earth=( 0.45 🌍 🌎 🌏 )
-clock=( 0.2 🕛 🕐 🕑 🕒 🕓 🕔 🕕 🕖 🕗 🕘 🕙 🕚 )
-moon=( 0.8 🌑 🌒 🌓 🌔 🌕 🌖 🌗 🌘 )
-orange_pulse=( 0.35 🔸 🔶 🟠 🟠 🔶 )
-blue_pulse=( 0.35 🔹 🔷 🔵 🔵 🔷 )
-football=( 0.25 ' 👧⚽️       👦' '👧  ⚽️      👦' '👧   ⚽️     👦' '👧    ⚽️    👦' '👧     ⚽️   👦' '👧      ⚽️  👦' '👧       ⚽️👦 ' '👧      ⚽️  👦' '👧     ⚽️   👦' '👧    ⚽️    👦' '👧   ⚽️     👦' '👧  ⚽️      👦' )
-blink=( 0.25 😐 😐 😐 😐 😐 😐 😐 😐 😐 😑 )
-camera=( 0.1 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📷 📸 📷 📸 )
-sparkling_camera=( 0.1 '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📷 ' '📸✨' '📷 ' '📸✨' )
-sick=( 0.9 🤢 🤢 🤮 )
-monkey=( 0.4 🙉 🙈 🙊 🙈 )
-bomb=( 0.25 '💣   ' ' 💣  ' '  💣 ' '   💣' '   💣' '   💣' '   💣' '   💣' '   💥' '    ' '    ' )
-
-###################################### 2 ######################################
-##### COPY THE CODE BELOW IN YOUR SCRIPT, RIGHT AFTER THE ANIMATIONS LIST #####
-###############################################################################
-############## If you would prefer to use a more compact version ##############
-################# of this code, go to the bottom of the file. #################
-###############################################################################
+declare BLA_classic_utf8=( 0.25 '—' "\\" '|' '/' )
```

Comment:
I changed the quoting on backslash so that [shellcheck](https://www.shellcheck.net/) stopped complaining. That's also what line 6 is for; shellcheck thinks that it is a problem that none of the animation variables are not used in this script, so I just shut that check off.

Reply:
Is there a specific reason you added `declare` to all the animations?

Reply:
It helps shellcheck do its job. Plus it might help with scope clashes if you source this in. I try to follow https://google.github.io/styleguide/shellguide.html plus carry over what I find are good habits from other languages and I find that using `declare` makes things clearer when I read them. It won't hurt, it might help.

Reply:
Understood.
It's a bit too verbose in my opinion, and in this case, it makes it harder to visualise the long animations.
I'll remove them for now as ShellCheck is doing fine without them, and the probabilities that they clash with other variables/arrays are virtually zero since you added the `BLA_` prefix.